src/interceptor.ts

```diff
@@ -225,6 +222,23 @@ export default class Interceptor extends EventEmitter implements IInterceptEvent
           responseSerializer,
         });
       });
+
```

Comment:
I thought we should move the pipeline call until after all the event listeners are attached.